resources/views/editUserProfile.blade.php

```diff
@@ -0,0 +1,101 @@
+<div class="ui form error">
+	<!-- User image -->
+	<div id ="userImage" class="field">
+		<label for="userImage">Photo</label>
+		<div class="userImageUploaderContainer">
+			<div class="imagePreviewContainer">
+				<img id="userImagePreview" src="https://lorempixel.com/400/400" alt="User image" class="ui small circular image preview"/>
+			</div>
+			<a href="#" class="imgUploader">
+				<button id="uploadImage" class="ui button primary">Upload image</button>
+			</a>
+		</div>
+		<input id="imgInput" type="file" name="userImage" style="display:none" multiple accept="image/x-png,image/jpeg">
+	</div>
+	<!-- User name -->
+	<div id ="userName" class="field">
+		<label for="userName">Name</label>
+		<input type="text" name="userName" placeholder="John Doe" value="Katie Arriaga">
+	</div>
+	<!-- User email -->
+	<div id="userEmail" class="field">
+		<label for="userEmail">E-mail</label>
+		<input type="email" name="userEmail" placeholder="john@example.com" value="katie@example.com">
+	</div>
+	<!-- Skillset -->
+	<div id="skillsets" class="field">
+		<label for="skillsets">Skills</label>
+		<select class="ui fluid search dropdown" multiple="">
+			<option value="">Skills</option>
+			<option value="angular">Angular</option>
+			<option value="css">CSS</option>
+			<option value="design">Graphic Design</option>
+			<option value="ember">Ember</option>
+			<option value="html">HTML</option>
+			<option value="ia">Information Architecture</option>
+			<option value="javascript">Javascript</option>
+			<option value="mech">Mechanical Engineering</option>
+			<option value="meteor">Meteor</option>
+			<option value="node">NodeJS</option>
+			<option value="plumbing">Plumbing</option>
+			<option value="python">Python</option>
+			<option value="rails">Rails</option>
+			<option value="react">React</option>
+			<option value="repair">Kitchen Repair</option>
+			<option value="ruby">Ruby</option>
+			<option value="ui">UI Design</option>
+			<option value="ux">User Experience</option>
+		</select>
+	</div>
+	<!-- Error message -->
+	<div id="errorMessage" class="ui error message">
+		<div class="header">Whoops! Something went wrong.</div>
+		<p>Please make sure to properly fill out all required fields.</p>
+	</div>
+	<!-- Save button -->
+	<button id="saveProfile" class="ui button primary">Save profile</button>
+</div>
+@section('scripts')
+<!-- Internal jQuery -->
+<script>
+	/* Search for Skillsets */
+	var skillsets = [
+		{ title: 'Skillset 1' },
+		{ title: 'Skillset 2' },
+		{ title: 'Skillset 3' },
+		{ title: 'Skillset 4' },
+		{ title: 'Skillset 5' },
+	];
+	$('.ui.fluid.search.dropdown')
+  		.dropdown();
+	$('.ui.search').search({source: skillsets});
+	$('.imgUploader').click(function(event) {
+		event.preventDefault();
+		$('#imgInput').click();
+	});
+
+	function readURL(input) {
+        if (input.files && input.files[0]) {
+            var reader = new FileReader();
+
+            reader.onload = function (e) {
```

Comment:
¿Qué haces si la lectura falla?

Reply:
Ahorita lo puse para que despliegue un alert. Creo que esto lo podemos revisitar cuando alguien tenga el user story de editar el perfil del usuario, if that's okay.